expyfun/_experiment_controller.py

```diff
@@ -940,6 +944,8 @@ def get_presses(self, live_keys=None, timestamp=True, relative_to=None):
             If timestamp==False, returns a list of strings indicating which
             keys were pressed. Otherwise, returns a list of tuples
             (str, float) of keys and their timestamps.
+            If releases is True, the last value in the tuple will be a string,
+            either 'press' or 'release'.
```

Comment:
@Eric89GXL what do you think about this?  I think putting `'press'` or `'release'` inside the return tuple is the right way to handle it, but I'm iffy on whether to have it only included when `releases=True`, versus always including it (so that all tuples would be `(<key>, <time>, 'press')` if `releases=False`).